### PR TITLE
Don't warn about potentially malformed pack mcmeta files in datagen

### PR DESCRIPTION
**Pull request by pupnewfster**
Datagen does not care whether pack.mcmeta files actually exist or not so if they happen to be malformed is irrelevant. So if the json is malformed due to having variables for replacement, just silence the error logging during datagen. https://github.com/neoforged/MDK/issues/9

### CODE REVIEW

**Comment by Technici4n** [comment]
Wouldn't running in dev print the same warning? Why limit the fix to datagen... :wink:

**Comment by pupnewfster** [comment]
It doesn't because dev runs against the files processed by processResources and the like. The reason for datagen this doesn't happen is that as a direct folder is specified as an argument to "existing" so that folder is queried as opposed to whatever gradle/IDE folder that is used while running for the active resources (to avoid situations like running datagen twice having the first run's output included in "existing" files.)

So basically if it is malformed in dev that means it will also be malformed in prod at which point minecraft will yell at you and we would rather devs fix bugs than lead to even more random error/warning log spam.

**Comment by Technici4n** [comment]
Ah I see, makes sense

**Review by marchermans — APPROVED** [review]
